app/static/playwright.config.ts

```diff
@@ -4,7 +4,9 @@ const config: PlaywrightTestConfig = {
     testMatch: '*.etest.ts',
     use: {
         baseURL: "http://localhost:3000"
-    }
+    },
+    workers: 1,
```

Comment:
Was getting sporadic browser test failures on CI only on first couple of tests, and allowing one retry seems to be enough to allow them all to pass - this seemed preferable to bumping up the timeout. Also specifically setting workers to 1 here, as this is how it runs in the github action.

Reply:
Doesn't this mean it will take longer to complete?